packages/ui/src/working-groups/queries/workingGroups.graphql

```diff
@@ -28,7 +28,10 @@ fragment WorkerFields on Worker {
   }
   application {
     id
-    openingId
+    opening {
+      id
+      unstakingPeriod
+    }
```

Comment:
Maybe fetching an opening in the modal would be better - we don't always need unstacking period.